src/mediaelement.js

```diff
@@ -391,6 +403,13 @@ export default class MediaElement extends WebAudio {
     destroy() {
         this.pause();
         this.unAll();
+        this.destroyed = true;
```

Comment:
should we put this at start of the block?

Reply:
no, it should be set after `unAll` is called.